WalletWasabi.Fluent/Views/MainWindow.axaml.cs

```diff
@@ -9,6 +10,7 @@ public class MainWindow : Window
 	public MainWindow()
 	{
 		InitializeComponent();
+		ExtendClientAreaChromeHints |= ExtendClientAreaChromeHints.OSXThickTitleBar;
```

Comment:
🚨MVVM Alert 🚨

Reply:
should be in XAML? why code behind?

Reply:
this is not possible to set in XAML.

Reply:
@soosr this is "View" related code. It will violate MVVM if i added that in any VM.

The purpose of that setting is to align the traffic buttons in macOS to line up with the acrylic trim.

Reply:
Solution here https://stackoverflow.com/questions/7718928/setting-multiple-enum-flags-in-xaml